rules/xcodeproj.bzl

```diff
@@ -25,13 +25,20 @@ def _xcodeproj_aspect_impl(target, ctx):
     deps += getattr(ctx.rule.attr, "deps", [])
     deps += getattr(ctx.rule.attr, "infoplists", [])
     deps.append(getattr(ctx.rule.attr, "entitlements", None))
+    deps.append(getattr(ctx.rule.attr, "test_host", None))
 
     # TODO: handle apple_resource_bundle targets
     test_env_vars = ()
     test_commandline_args = ()
+    bazel_build_target_name = ""
+    if target.label.workspace_name != "":
```

Comment:
@amberdixon what if we can make code reading easier by having example here about what each var might represent in real life (as comments), so that understanding of code can be easier in future?